cmd/verify/verify.go

```diff
@@ -15,23 +15,25 @@ import (
 
 	"github.com/elixir-oslo/crypt4gh/keys"
 	"github.com/elixir-oslo/crypt4gh/streaming"
+	"github.com/xeipuuv/gojsonschema"
 
 	log "github.com/sirupsen/logrus"
 )
 
 // Message struct that holds the json message data
 type Message struct {
-	Filepath     string `json:"filepath"`
+	Filepath     string `json:"filep_ath"`
```

Comment:
Typo?

Reply:
Good catch, but this is moot anyway since it's filepath now.